pybliometrics/scopus/superclasses/search.py

```diff
@@ -67,9 +67,7 @@ def __init__(self, query, api, refresh, view='STANDARD', count=200,
         else:
             name = ''
```

Comment:
Due to the new line 70 this is not now superfluous

Reply:
Did you mean "this is now superfluous"?

Reply:
Yes. It's not necessary to instantiate a str variable any longer. You used this variable to append to, but now the variable is created in one go